react-client/src/components/card/PetCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { Icon } from "../icon/Icon";
+import Card from "./Card";
+import { classNames } from "~/util/styleUtil";
+
+interface IPetCardProps {
```

Comment:
Prefer using type instead of interface, and the `I` is unnecessary.
```suggestion
type PetCardProps = {
```